.github/workflows/publish-release.yml

```diff
@@ -61,11 +68,17 @@ jobs:
         uses: olegtarasov/get-tag@v2.1
         id: tagName
 
+      - name: Load secrets from 1Password
+        uses: 1password/load-secrets-action@v2
+        with:
+          export-env: true
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+          SENTRY_AUTH_TOKEN: op://ddev-intellij-plugin-secrets/sentry/credential
+          SENTRY_ORG: op://ddev-intellij-plugin-secrets/sentry/org
```

Comment:
I'm pretty sure this should be in the github environment variables, since it's not a secret.